server/model-config.json

```diff
@@ -0,0 +1,69 @@
+{
+  "_meta": {
+    "sources": [
+      "loopback/common/models",
+      "loopback/server/models",
+      "../common/models",
+      "./models"
```

Comment:
We do have both currently since we have `Service` and `Action` in `common/models`, extended as `ServerService` and `InstanceAction` in the `./models`. I had fixed this on the 3.x PM and 4.x mesh-models branch to only use `Service` and `Action` without having to extend them. However that requires a API rev so I kept it out of this PR.